importers/houzz.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/zxsecurity/steamer/importers/util"
+)
+
+type TemplateLineParser struct{}
+
+func (t TemplateLineParser) ParseLine(line string) ([]interface{}, error) {
+	entries := make([]interface{}, 0)
+	// split a line in the text file
+	data := util.SplitString(line, ' ', true, true)
+	// check data here
+	if len(data) > 2 || strings.Contains(data[3], "SEC"{
```

Comment:
I think you're missing a `)` here

Reply:
Oh cool I didn't even know that existed. Will fix it now :)